docs/signature-layout.md

```diff
@@ -0,0 +1,29 @@
+# Signature File Layout
+
+A common file layout for storing and serving signatures provides a consistent way to reference image signatures. Signatures on a filesystem or a web server shall use this common layout. Signatures stored in a REST API are not required to use this common layout.
+
+## Specification
+
+**SCHEME://[URI]/PATH_PREFIX/REGISTRY/REPOSITORY/IMAGE@MANIFEST_DIGEST/signature-INT**
+
+* **SCHEME**: The transport scheme, e.g. **file://** or **https://**
+* **URI**: For remote transport schemes, the signature server URI, e.g. **example.com**
+* **PATH_PREFIX**: The path to the base of the registry directory
+* **REGISTRY**: The registry URI and optional port, e.g. **registry.example.com:5000**
+* **REPOSITORY**: The repository namespace. May occur multiple times for registries that support multiple repository namespaces.
+* **IMAGE**: The name of the image
```

Comment:
The paths for lookaside are `"`$root`/`${ref`.FullName()`}`@`$manifestDigest`/signature-`$index`"` .  They haven’t been designed to be necessarily equivalent to any URLs in the v2 docker/distribution API.

That said, AFAICT the <name> component of `FullName` (used by lookaside) and `RemoteName` (used by docker/distribution) are equal, that’s not the issue; the unreconcilable divergence is that the lookaside prefixes that by hostname:port, and docker/distribution by `/v2/`